src/maxwell-web-view.js

```diff
@@ -23,66 +23,54 @@
 
 (function() {
 
-function throttle (func, limit) {
-    let stamp, id;
+let children = new Array();  /* List of children */
+let children_hash = {};      /* Hash table of children */
 
-    return () => {
-        let self = this;
-        let args = arguments;
+function update_position_size () {
+    let positions = null;
 
-        if (!stamp) {
-            func.apply(self, args);
-            stamp = Date.now();
-            return;
-        }
-
-        clearTimeout(id);
-        id = setTimeout(() => {
-            let now = Date.now();
-            if (now - stamp >= limit) {
-                func.apply(self, args);
-                stamp = now;
-            }
-        }, limit - (Date.now() - stamp));
-    }
-}
-
-function update_positions () {
-    let children = window.maxwell.children;
-    let positions = new Array ();
-
-    for (let id in children) {
-        let child = children[id];
+    for (let i = 0, len = children.length; i < len; i++) {
```

Comment:
I did not used forEach() because I am trying to make this function as fast as possible, and I assume forEach() implies a function call for each iteration which I guess its slower than code in a for loop